server/modules/category/api/categoryApi.js

```diff
@@ -101,22 +101,6 @@ export const init = (app) => {
 
   // ==== READ
 
-  app.get('/survey/:surveyId/categoriesAll', AuthMiddleware.requireSurveyViewPermission, async (req, res, next) => {
-    try {
-      const { surveyId, draft, validate } = Request.getParams(req)
-
-      const categories = await CategoryService.fetchCategoriesAndLevelsBySurveyId({
```

Comment:
can u check if CategoryService.fetchCategoriesAndLevelsBySurveyId is still used ?